Doc/tutorial/introduction.rst

```diff
@@ -478,73 +478,123 @@ example::
 First Steps Towards Programming
 ===============================
 
-Of course, we can use Python for more complicated tasks than adding two and two
-together.  For instance, we can write an initial sub-sequence of the
-`Fibonacci series <https://en.wikipedia.org/wiki/Fibonacci_number>`_
-as follows::
-
-   >>> # Fibonacci series:
-   ... # the sum of two elements defines the next
-   ... a, b = 0, 1
-   >>> while a < 10:
-   ...     print(a)
-   ...     a, b = b, a+b
-   ...
-   0
+We can use Python for more complex tasks than adding two and two together.
+For instance, we can compute the numbers of the
+`Fibonacci series <https://en.wikipedia.org/wiki/Fibonacci_number>`_.
+To do that we need to utilize three following concepts.
+
+Multiple Assignment
+-------------------
+
+A *multiple assignment* of variables us allows to set values of more than one
+variable on a single line.  Notice that if the value assignment is a
+expression, it is first evaluated and then assigned::
+
+   >>> a, b = 1, 5  # multiple assignment of two variables
+   >>> a
    1
+   >>> b
+   5
+   >>> a, b = b, a + b  # first a is set to value of b, then b is set to sum of a and b
+   >>> a
+   5
+   >>> b
+   6
+
+Repeating our code
+------------------
+
+The :keyword:`while` loop is example of a repeating cycle. It performs
+a block of code continuously as long as the condition is
+fulfilled.  The test used in the example is a simple comparison you might
+know from arithmetics.  Fulfilled conditions have the value ``True``.
+Unfulfilled conditions have the value ``False``::
+
+   >>> 1 < 3  # 1 is less than 3
+   True
+   >>> 'king' == "king"  # the strings are equal
+   True
+   >>> 4 >= 6  # 4 is not greater than or equal to 6
+   False
+
+In Python, the following comparison operators are used:
+
+  * ``<``: Less than
+  * ``>``: Greater than
+  * ``==``: Equal to
+  * ``<=``: Less than or equal to
+  * ``>=``: Greater than or equal to
+  * ``!=``: Not equal to
+
+Let's create a simple loop.  We need the keyword ``while`` and a condition;
+in this case, ``count < 5``.  To ensure that the loop finishes, we must make
+sure that the condition is *not fulfilled* at a certain step.  Otherwise, the
+code would repeat indefinitely.  To achieve that, we can increase the value
+of count.  As soon as the variable ``count`` reaches the value 5, the
+condition will be ``False``::
+
+   >>> count = 0;  # define variable to which we will be adding 1 in a loop
+   >>> while count < 5: count = count + 1  # hit enter one more time to start the loop
+   ...
+   >>> count  # when count reached value of 5, while loop finished
+   5
+
+Note that block inside the ``while`` loop, or *body* of the loop is *indented*.
```

Comment:
The example above uses a one-liner: `while count < 5: count = count + 1`

Let's use indentation in the example, to match the text. Something like this?


```suggestion
   >>> while count < 5:
   ...    count = count + 1  # hit enter one more time to start the loop
   ...
   >>> count  # when count reached value of 5, while loop finished
   5

Note that block inside the ``while`` loop, or *body* of the loop is *indented*.
```